src/tm/class-wpml-tm-page-builders-hooks.php

```diff
@@ -49,15 +49,19 @@ public function pro_translation_completed_action( $new_post_id, array $fields, s
 	}
 
 	/**
-	 * @param array    $fields
-	 * @param stdClass $job
+	 * Filter translation fields.
+	 *
+	 * @param array    $fields Translation fields.
+	 * @param stdClass $job    Translation job.
 	 *
 	 * @return array
 	 */
 	public function adjust_translation_fields_filter( array $fields, $job ) {
 		$worker = $this->get_worker();
 
-		return $worker->adjust_translation_fields_filter( $fields, $job );
+		$fields = $worker->adjust_translation_fields_filter( $fields, $job );
+
+		return $fields;
```

Comment:
@kagg-design This change is not really needed.

Reply:
@brucepearson yes, it is leftover from previous modifications. I have removed it. Could you approve last commit?